src/server/svr_func.c

```diff
@@ -5460,8 +5460,9 @@ prov_request_deferred(struct work_task *wtask)
 		if (exit_status == APP_PROV_SUCCESS &&
 			(pnode->nd_state & INUSE_DOWN))
 			set_vnode_state(pnode, ~INUSE_DOWN, Nd_State_And);
-		else
-			is_vnode_prov_done(pnode->nd_name);
+
+		is_vnode_prov_done(pnode->nd_name);
```

Comment:
The description of the PR made it sound like this call was being moved.  I don't see it removed from its previous location.  Was that done previously?

Reply:
Yes, it has done in the last PR of this ticket.